CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 2018.06.1-dev 
+## 2018.08.0
```

Comment:
Can you add the link of the [PR](https://github.com/aws/amazon-ecs-cni-plugins/pull/81).